integration/Substra/plot_substrafl_flamby.py

```diff
@@ -0,0 +1,446 @@
+#!/usr/bin/env python
```

Comment:
Can you:
- convert this file to markdown
- change its name to `flamby-integration-into-substra.md` to comply with flamby's requirements ?